### PR TITLE
PR - Doc: update readme

### PR DESCRIPTION
**Pull request by yvalentin**
### FYI 
This branch come from `feature/update-node` [branch](https://github.com/betagouv/transition-ecologique-entreprises-widget/tree/feature/update-node)  
So feature/update-node` need to be merge before ;-)

### What's done
- A menu has been add at the beginning of the readme
- The main screenshot has been updated to correspond to the main page of the current version
- A `quick start` part has been added to give command to just start the project quickly on local machine
- Some commands have been corrected
- A new part `Development mode` has been added to give command to run the project has a developer and work on the project
- Credits part has been updated
- main `npm` commands have been update

### CODE REVIEW

File: packages/web/package.json
Comment (JulienParis) [inline packages/web/package.json:1]: Isn't it redondant with #167 ?

File: packages/web/.env.example [hunk 1]
```diff
@@ -0,0 +1,11 @@
+VITE_NO_DEBUG_SWITCH=false
+VITE_DATA_DIR_PATH=../data/programs
+VITE_DEPLOY_URL=https://tee-frontend.osc-fr1.scalingo.io
+
+VITE_BREVO_SENDER_EMAIL=france-transition@beta.gouv.fr
```
Comment (JulienParis) [inline packages/web/.env.example:5]: There is no need for this line anymore

File: packages/backend/package.json
Comment (JulienParis) [inline packages/backend/package.json:1]: Isn't it redundant with #167 ?

File: package.json [hunk 2]
```diff
@@ -10,22 +10,26 @@
...
+    "dev-front": "npx concurrently \"npm run build-json-output-watch\" \"npm run dev -w packages/web\"",
     "test-data": "npm run test -w packages/data"
   },
   "workspaces": [
     "./packages/*"
   ],
   "engines": {
-    "node": "16.12.0",
-    "npm": "8.1.0"
+    "node": "~18.18",
```
Comment (JulienParis) [inline package.json:31]: Isn't it redundant with #167 ?

File: README.md [hunk 3]
```diff
@@ -30,32 +45,56 @@ height="75px">
...
-![](./packages/web/images/screenshots/screenshot-230602-a.png)
-
+![](./packages/web/images/screenshots/screenshot-2023-10-19.png)
 
 ## Project Setup
 
+### Stack
+- [NodeJS](https://nodejs.org/) (v18)
+- Back: [Express](http://expressjs.com/) (v4)
+- Front: [VueJS](https://vuejs.org/) (v3)
```
Comment (JulienParis) [inline README.md:55]: Perhaps we could mention the DSFR package here, just make it more visible from the start ? 

```md
- Design system: [DSFR](https://www.systeme-de-design.gouv.fr/)
```

File: .nvmrc
Comment (JulienParis) [inline .nvmrc:1]: Isn't it redundant with #167 ?

File: .github/workflows/tests.yml
Comment (JulienParis) [inline .github/workflows/tests.yml:1]: Isn't it redundant with #167 ?

**Review by JulienParis — CHANGES_REQUESTED** [review]

**Comment by JulienParis** [comment]
praise: nice PR ! the `start-*` scripts are quite a bonus for people just wanting to test without having to play w/ `.env` files. I noted the branch `feature/update-node` should be merged first, I just noted all the comments about #167 as questions (just beginning to adopt [conventional comments](https://conventionalcomments.org/) )

File: README.md [hunk 4]
```diff
@@ -30,32 +45,56 @@ height="75px">
...
+
+### Description
 This repository is a monorepo, based on [npm 
 workspaces](https://docs.npmjs.com/cli/v7/using-npm/workspaces/) composed of 
 two workspaces:
 
 - the ["web" workspace](./packages/web/README.md) contains the front-end implementations.
 - the ["backend" workspace](./packages/backend/README.md) contains the back-end implementations.
```
Comment (JulienParis) [inline README.md:63]: todo: I just saw this this morning, the `data` workspace information is missing =>

```md
- the ["data" workspace](./packages/data/README.md) contains the datasets and common files for datasets builds.
```

**Review by JulienParis — CHANGES_REQUESTED** [review]

File: packages/web/vite.config.ts
[the remark this answers is not in the document]
Comment (yvalentin) [reply packages/web/vite.config.ts:1]: I have done a rollback on this

**Pull request merged**